x-pack/plugins/security_solution/cypress/integration/timelines/row_renderers.spec.ts

```diff
@@ -84,6 +85,9 @@ describe('Row renderers', () => {
     cy.get(TIMELINE_ROW_RENDERERS_DISABLE_ALL_BTN).should('exist');
     cy.get(TIMELINE_ROW_RENDERERS_MODAL_ITEMS_CHECKBOX).should('be.checked');
 
+    // Intercepts should be before click handlers that activate them rather than afterwards or you have race conditions
```

Comment:
I think I ran into this before and I got in the habit of just lifting these to the top (assuming that you aren't doing anything beforehand to activate them multiple times).

Reply:
I tried the very top as well as `beforeEach` but it does some other networking so this was the sweet spot for it. I do agree, anytime we can put them at the top of the test or in a `beforeEach`, those are the best spots for 'em.

Reply:
>(assuming that you aren't doing anything beforehand to activate them multiple times)

Yes! Exactly. This test was. All good advice and good practices it feels like. I think this one was just an oversight/oppsie and it ended up after the button clicks instead of before.